src/main.py

```
@@ -12,20 +11,21 @@
 
 console = Console()
 
+
 async def main():
```

Ruff (0.8.0)

`main` is too complex (19 > 10)

(C901)